tests/core/test_cost_calculation.py

```diff
@@ -107,6 +107,18 @@ async def test_strict_mode(self):
         coin_name = npc_list[0].coin_name
         error, puzzle, solution = get_puzzle_and_solution_for_coin(program, coin_name)
 
+    async def test_clvm_strict_mode(self):
+        program = SerializedProgram.from_bytes(
+            binutils.assemble(
+                "(i (a (q 0xfe) (q ())) (q ()) (q ((51 0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 200))))"
```

Comment:
the subexpression `(a (q 0xfe))` executes the unknown opcode `0xfe` which is a fatal error in safe mode, and returns `()` (i.e. false) in normal mode